src/api/public/apidocs-new/paths/source_project_name_delete.yaml

```diff
@@ -0,0 +1,28 @@
+delete:
+  summary: Delete a specified project
+  description: |
+    Deletes a specified project and all the packages of this project
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: force
+      in: query
+      schema:
+        type: integer
+      description: |
+        If 1, the project is deleted even if the repositories of other
+        projects include a path to this project. After deletion, the path
+        in the other repository will start pointing towards 'deleted/standard' to
+        prevent the build and publishing
+    - name: comment
+      in: query
+      schema:
+        type: string
+  responses:
```

Comment:
I miss at least one example of a 403 response here. It can be raised in [this line of code](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/source_project_controller.rb#L62).

Reply:
Thanks for checking it out. I've documented 403